src/Pelagos/Bundle/AppBundle/Form/DatasetSubmissionType.php

```diff
@@ -262,6 +262,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'delete_empty' => true,
                 'required' => true,
             ))
+            ->add('erddapUrl', Type\TextType::class, array(
+                'label' => 'ERDDAP Url',
+                'required' => false,
+                'attr' => array('data-rule-url' => true),
```

Comment:
What does this do?

Reply:
It is a jquery validate plugin for html 5 data attributes

Reply:
I guess I should have known that.